engines/freescape/gfx.cpp

```diff
@@ -0,0 +1,117 @@
+/* ScummVM - Graphic Adventure Engine
+ *
+ * ScummVM is the legal property of its developers, whose names
+ * are too numerous to list here. Please refer to the COPYRIGHT
+ * file distributed with this source distribution.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+#include "gui/message.h"
+#include "graphics/renderer.h"
+#include "engines/util.h"
+
+#include "freescape/gfx.h"
+
+namespace Freescape {
+
+Renderer::Renderer(OSystem *system, int screenW, int screenH, Common::RenderMode renderMode)
+	: _system(system) {
```

Comment:
I do not see you using `_system` in this class. And in general, it is customary to simplify use `g_system` where it is needed.

Reply:
Removed!